third_party/proxy/meta_proxier.go

```diff
@@ -160,12 +161,12 @@ func (proxier *metaProxier) OnEndpointsSynced() {
 }
 
 func (proxier *metaProxier) GetServiceByIP(serviceStr string) (ServicePortName, bool) {
```

Comment:
Hi @jianjuns ,
`GetServiceByIP` is a method in Proxier interface defined in pkg/third_party/proxy
This was originally defined in pkg/agent/proxy but moved to third-party pkg. This is currently used by Flow Exporter running in Agent and no one else uses it. 
Therefore, I think it should follow the placement of `GetServiceFlowKeys` method and placed in pkg/agent/proxy. Do you agree?

Reply:
That sounds good to me.

Reply:
Thanks Srikar and Jianjun, moved the GetServiceByIP under pkg/agent/proxy and removed it from Provider interface in third_party/proxy.